packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -1116,7 +1116,7 @@ export class KubernetesClient {
    */
   async applyResourcesFromYAML(context: string, yaml: string): Promise<KubernetesObject[]> {
     const manifests = await this.loadManifestsFromYAML(yaml);
-    return this.applyResources(context, manifests, 'apply');
+    return this.applyResources(context, manifests);
```

Comment:
It seems by reading the code that if you don't pass the `namespace` value, the action will be done in the `default` namespace. Have you tested in a non-`default` namespace?

Reply:
https://github.com/containers/podman-desktop/assets/6422176/5f3255de-6f03-4499-8abd-fdea16019e33

I realized I uploaded the wrong file, but the same still applies (editing yaml)